lib/src/client.dart

```diff
@@ -1223,206 +1222,15 @@ class Client extends MatrixApi {
   /// repository APIs, for example, proxies may enforce a lower upload size limit
   /// than is advertised by the server on this endpoint.
   @override
-  Future<ServerConfig> getConfig() =>
-      _serverConfigCache.fetch(() => _getAuthenticatedConfig());
-
-  // TODO: remove once we are able to autogen this
-  Future<ServerConfig> _getAuthenticatedConfig() async {
-    String path;
-    if (await authenticatedMediaSupported()) {
-      path = '_matrix/client/v1/media/config';
-    } else {
-      path = '_matrix/media/v3/config';
-    }
-    final requestUri = Uri(path: path);
-    final request = http.Request('GET', baseUri!.resolveUri(requestUri));
-    request.headers['authorization'] = 'Bearer ${bearerToken!}';
-    final response = await httpClient.send(request);
-    final responseBody = await response.stream.toBytes();
-    if (response.statusCode != 200) unexpectedResponse(response, responseBody);
-    final responseString = utf8.decode(responseBody);
-    final json = jsonDecode(responseString);
-    return ServerConfig.fromJson(json as Map<String, Object?>);
-  }
-
-  ///
-  ///
-  /// [serverName] The server name from the `mxc://` URI (the authoritory component)
-  ///
-  ///
-  /// [mediaId] The media ID from the `mxc://` URI (the path component)
-  ///
-  ///
-  /// [allowRemote] Indicates to the server that it should not attempt to fetch the media if it is deemed
-  /// remote. This is to prevent routing loops where the server contacts itself. Defaults to
-  /// true if not provided.
-  ///
-  @override
-  // TODO: remove once we are able to autogen this
-  Future<FileResponse> getContent(String serverName, String mediaId,
```

Comment:
Doesn't this break TI Messenger compatibility as well as compatibility with older servers?

Reply:
hmm, it's a breaking change and we should really remove these asap. I think we already decided to not autogen these?

Reply:
hm breaking change, but the older servers part makes sense brb

Reply:
It is fine to break the API, it is not really okay to just have no media support now on TIM and other older servers. That would affect our users as well.